src/autopas/pairwiseFunctors/CellFunctor.h

```diff
@@ -97,13 +105,24 @@ class CellFunctor {
 
   const double _sortingCutoff;
 
+  /**
+   * This value is used to switch on and off the sorting functionality of the CellFunctor. Sorting is enabled by default
+   */
+  bool _useSorting{true};
+
   /**
    * Min. number of particles to start sorting.
-   * @todo Currently, this is disabled because of https://github.com/AutoPas/AutoPas/issues/418
    */
-  constexpr static unsigned long _startSorting = std::numeric_limits<unsigned long>::max();
+  constexpr static unsigned long _sortingThreshold = 8;
```

Comment:
Add a link to this PR to show how we came up with the number
https://github.com/AutoPas/AutoPas/pull/619

Reply:
doen in [9b3ad7d](https://github.com/AutoPas/AutoPas/pull/619/commits/9b3ad7dcc62319f6afd35d92288ce888e38eac12)